scripts/windows/README.md

```diff
@@ -0,0 +1,63 @@
+# Windows build scripts
+
+Originally used only for GitHub actions, these scripts can help windows users to be able to build locally.
+While still very new (and open to feedback and [pull requests](https://github.com/php/php-src/fork))
+
+## Pre-requisites
+
+- Windows 2019 or newer [Microsoft](https://www.microsoft.com/en-us/evalcenter/download-windows-server-2019)
+- Administrative access
+- Chocolatey installed to use as a package manager [Install instructions](https://chocolatey.org/install)
+- MySQL installed [see our CI setup](.github/actions/setup-windows/action.yml)
+- SQL Server Installed [see our CI setup](.github/actions/setup-windows/action.yml)
+- Postgres Installed [see our CI setup](.github/actions/setup-windows/action.yml)
+- Git installed
+- Unzip utility installed
+- 7zip utility installed
+- Mirosoft Visual Studio 2019 Community or newer [Microsoft](https://visualstudio.microsoft.com/vs/older-downloads/)
+
+## Intended order of operations
+
+1. Setup pre-requisites
+2. Setup environment [env.bat](./scripts/windows/env.bat)
+3. Build PHP [build.bat](./scripts/windows/build.bat)
+4. Run PHP Tests [test.bat](./scripts/windows/test.bat)
+5. Cleanup [cleanup.bat](./scripts/windows/cleanup.bat)
+
+## env.bat
+
+The purpose of this script is simply to setup environment. It is intended to be run via command-line `cmd` under windows as an administrator.
+You can adjust values in this script to:
```

Comment:
We shouldn't encourage editing committed files. Why can't these values be set using actual env variables?

Reply:
The CI edits committed files today. Did you not know this? Did nobody tell you? Is there a plan to first change that? I can wait.

Reply:
CI is never going to create new commits based on the edited files, but users developing/compiling locally might accidentally include edits to build files in unrelated commits. An env var (and/or command line arg?) would help to mitigate this problem.

Reply:
This isn't creating new commits @DaveRandom the script has always edited `config.inc` (maybe other files). If that edit (`echo "details" >> file`) is run twice, outside of CI, which checks out a fresh copy of the file, then multiple lines are edited into that file. The git reset and editing the env.bat are just ways for folks to run the build with other settings. Technically more file-based edits are needed to build different PHP versions if you go back too far (even 8.2, 8.1 from 8.3) then different extensions are exercised.